client/app/components/ui/required-asterisk.hbs

```diff
@@ -0,0 +1,6 @@
+<sup>
+  <FaIcon @icon="asterisk" class="text-red text-weight-normal" @transform="shrink-4" />
```

Comment:
Is this icon accessible? Should we add a hidden * screen readers?

Reply:
I'll switch to using `<LabsUi::IconTooltip />` and it will be accessible. Screen readers can read the tooltip text in the Ember Tooltip.